python/gui/auto_generated/editorwidgets/qgsdatetimeedit.sip.in

```diff
@@ -58,17 +59,31 @@ setDateTime set the date time in the widget and handles null date times.
 
     QDateTime dateTime() const;
 %Docstring
-dateTime returns the date time which can eventually be a null date/time
+dateTime returns the date time which can be a null date/time
```

Comment:
This should just be "Returns the...." (Remove the leading "datetime")

Reply:
Done, I have done a review in the file.